app/cdash/include/CDash/Config.php

```diff
@@ -53,18 +60,19 @@ public function getProtocol(): string
     }
 
     /**
-     * @return string
+     * @deprecated 09/04/2023  Use url() instead.
      */
-    public function getServerPort()
+    public function getServerPort(): ?string
     {
         if (isset($_SERVER['SERVER_PORT'])
             && $_SERVER['SERVER_PORT'] != 80
             && $_SERVER['SERVER_PORT'] != 443) {
             return $_SERVER['SERVER_PORT'];
         }
+        return null;
     }
 
-    public function getPath(): string
+    private function getPath(): string
```

Comment:
Should this function also be marked as deprecated?

Reply:
It's now a private member, in a class where every other function is deprecated, so I thought that it would be overkill to mark it as deprecated.